src/index.ts

```diff
@@ -329,6 +336,17 @@ function ksmAddrDecoder(data: string): Buffer {
   return new Buffer(ss58Decode(data));
 }
 
+function filAddrEncoder(data: Buffer): string {
+  const address = filNewAddress('1', data)
```

Comment:
Pretty sure we shouldn't be hardcoding version 1 here. From looking at the library, it looks like the first byte in the binary format is the version byte, and we should probably be preserving that instead.